pkg/server/api/server_test.go

```diff
@@ -238,6 +238,33 @@ func TestGetAuthority(t *testing.T) {
 		"Request body from /authority unexpectedly returned '%s'", string(body))
 }
 
+func TestGetAuthorityWithExtraCAs(t *testing.T) {
+	c, err := generateContext(t)
+	require.NoError(t, err)
+	c = addExtraAuthorities(t, c)
+
+	req, err := generateHostRequest()
+	require.NoError(t, err, "Error reading test ssh key")
+
+	rec := httptest.NewRecorder()
+	c.Authority(rec, req)
+
+	rec.Flush()
+	require.Equalf(t, 200, rec.Code, "Request to /authority failed with %d", rec.Code)
+
+	body, _ := ioutil.ReadAll(rec.Body)
+	receivedCAs := strings.Split(string(body), "\n")
+	expectedCA1Bytes, err := ioutil.ReadFile("testdata/server_ca.pub")
```

Comment:
nit: `ioutil.ReadFile` is just `os.ReadFile` these days: https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/io/ioutil/ioutil.go;l=36

Reply:
Good point, there are a few more places we use `ioutil`, so probably best to address that in a follow-up PR. Also, we should ensure we're on latest Go everywhere, I can also follow up on that.